index.html

```diff
@@ -1256,8 +1256,96 @@
       <section>
         <h3>Usage Patterns</h3>
 
-        <p class="issue">
-When verifiable claims are used can be used for correlation purposes.
+        <p>
+Despite the best efforts to assure privacy, the actual use of verifiable
+claims can potentially lead to de-anonymization and a loss of privacy.
+This correlation can occur:
+        </p>
+
+        <ol>
+          <li>
+When the same claim is presented to the same inspector more than once –
+that inspector could infer that the holder is the same individual.
+          </li>
+          <li>
+When the same claim is presented to different inspectors, and either those
+inspectors collude or a third party has access to transaction records from
+both inspectors – the observant party could infer that the individual
+presenting the claims is the same person at both services, i.e., the
+accounts are controlled by the same person.
+          </li>
+          <li>
+When the same subject ID of a claim refers to the same subject across
```

Comment:
I wonder if this should be a bullet item under "identifier-based" correlation. I'm on the fence. We could have a big section on correlation (all means), but I'm concerned that section would be huge. So, splitting it up seems good, except that we may have statements like "hey if you use the same subject id, you can be correlated" in multiple sections. We could point identifier-based correlation to the appropriate section. Keep this here for now, we can make a second pass at some point in the future.